project/AppDependencies.scala

```diff
@@ -1,58 +1,46 @@
-import sbt._
+import sbt.{Def, _}
 
 private object AppDependencies {
 
   import play.sbt.PlayImport._
-  import play.core.PlayVersion
 
-  private val playPartialsVersion = "8.3.0-play-28"
-  private val httpCachingClientVersion = "9.5.0-play-28"
-  private val flexmarkVersion = "0.19.1"
-  private val okHttpVersion = "3.9.1"
-  private val jsonEncryptionVersion = "5.1.0-play-28"
-  private val hmrcMongoVersion = "0.71.0"
-  private val domain = "8.1.0-play-28"
+  private final val playV = "play-30"
+  private final val playJsonV = "3.0.2"
 
-  val compile = Seq(
+  val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"       %% "domain"                        % domain,
-    "uk.gov.hmrc"       %% "play-partials"                 % playPartialsVersion,
-    "uk.gov.hmrc"       %% "http-caching-client"           % httpCachingClientVersion,
-    "uk.gov.hmrc"       %% "json-encryption"               % jsonEncryptionVersion,
-    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"            % hmrcMongoVersion,
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"    % "5.24.0",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-28"    % "8.4.0",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping" % "1.12.0-play-28",
-
-    "com.vladsch.flexmark" % "flexmark-all" % flexmarkVersion,
-    "com.beachape" %% "enumeratum-play" % "1.5.15",
-    "com.squareup.okhttp3" % "mockwebserver" % okHttpVersion,
-    "com.typesafe.play" %% "play-json" % "2.8.1",
-    "com.typesafe.play" %% "play-json-joda" % "2.8.1",
-    "org.typelevel"     %% "cats-core"      % "2.10.0",
-
-    compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.5" cross CrossVersion.full),
-    "com.github.ghik" % "silencer-lib" % "1.7.5" % Provided cross CrossVersion.full
+    "uk.gov.hmrc"          %% s"domain-$playV"                        % "9.0.0",
+    "uk.gov.hmrc"          %% s"play-partials-$playV"                 % "9.1.0",
+    "uk.gov.hmrc"          %% s"http-caching-client-$playV"           % "11.1.0",
+    "uk.gov.hmrc.mongo"    %% s"hmrc-mongo-$playV"                    % "1.4.0",
+    "uk.gov.hmrc"          %% s"bootstrap-frontend-$playV"            % "8.4.0",
+    "uk.gov.hmrc"          %% s"play-frontend-hmrc-$playV"            % "8.4.0",
+    "uk.gov.hmrc"          %% s"play-conditional-form-mapping-$playV" % "2.0.0",
+    "uk.gov.hmrc"          %% "json-encryption"                       % "5.1.0-play-28",
```

Comment:
Just wanting to double check why this isn't at Play 3? Is it not supported yet or is it going to be a massive breaking change, or something else?